docs/requirements.txt

```diff
@@ -1,2 +1,3 @@
 sphinx==3.5.4
--e git+git://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme 
+-e git+https://github.com/pytorch/pytorch_sphinx_theme.git@b4d0005#egg=pytorch_sphinx_theme 
```

Comment:
The reason why this is pinned in audio is that we customize the behavior of HTML so that tutorials have links like `Run in Google Colab`, `Download Notebook` and `View on GitHub`.

To achieve the same effect, you might also want to check-in https://github.com/pytorch/audio/blob/main/docs/source/_templates/layout.html with modification of `audio` -> `text`.

Reply:
I made the changes though I don't see those link in my local doc builds. Not sure if I am doing something wrong?

Reply:
okay, I did not test the tutorials without `_tutorial` suffix.

See https://github.com/pytorch/text/pull/1468#discussion_r782639168 for the fix.

Reply:
> See [#1468 (comment)](https://github.com/pytorch/text/pull/1468#discussion_r782639168) for the fix.

cool, this worked. thanks!